docs/next/modules/en/pages/getting-started/create-first-cluster/using_fleet.adoc

```diff
@@ -35,7 +35,7 @@ export CLUSTER_NAME=cluster1
 export CONTROL_PLANE_MACHINE_COUNT=1
 export WORKER_MACHINE_COUNT=1
 export KUBERNETES_VERSION=v1.30.0
-export NAMESPACE=default
+export NAMESPACE=capi-clusters
```

Comment:
Since there are a bunch of occurrences of namespace here, you could try this.
* At the top of the file, define a variable, e.g., `:namespace: capi-clusters`
* Now in all the code blocks, switch on the attribute replacement using `subs="attributes"`. E.g.: `[source,bash,subs="attributes"]`
* In the code block, use `{namespace}` instead of the actual name, like `default` or `capi-clusters`.

This will help define the variable at one place so that if in future someone wants to change the namespace, it only needs to be done at one place.

Reply:
I prefer to keep it as it is